plugins/plugins.go

```diff
@@ -395,8 +405,10 @@ func (m *Manager) Stop(ctx context.Context) {
 		}
 	}()
 
+	timeoutCtx, cancel := context.WithTimeout(ctx, time.Duration(m.gracefulShutdownPeriod)*time.Second)
```

Comment:
I'd propose we reuse the `ctx` variable, it's not at all uncommon:
```suggestion
	ctx, cancel := context.WithTimeout(ctx, time.Duration(m.gracefulShutdownPeriod)*time.Second)
```

Reply:
Thanks! Will change.